spec/javascript-spec.coffee

```diff
@@ -53,6 +53,28 @@ describe "Javascript grammar", ->
       {tokens} = grammar.tokenizeLine('with')
       expect(tokens[0]).toEqual value: 'with', scopes: ['source.js', 'keyword.control.js']
 
+    map =
```

Comment:
I like how the loop generates multiple tests because when tokenizing of one keyword breaks, only its test will fail and not the entire block. This was useful for me in development when I wrote these tests first and changed one by one regular expression to get them passing.